root/artist_credit/EntityList.js

```diff
@@ -0,0 +1,79 @@
+/*
+ * @flow strict-local
+ * Copyright (C) 2020 MetaBrainz Foundation
+ *
+ * This file is part of MusicBrainz, the open internet music database,
+ * and is licensed under the GPL version 2, or (at your option) any
+ * later version: http://www.gnu.org/licenses/gpl-2.0.txt
+ */
+
+import * as React from 'react';
+
+import PaginatedResults from '../components/PaginatedResults';
+import EntityLink
+  from '../static/scripts/common/components/EntityLink';
+import expand2text from '../static/scripts/common/i18n/expand2text';
+
+import ArtistCreditLayout from './ArtistCreditLayout';
+
+const headingsText = {
+  recording: N_ln('{num} recording found', '{num} recordings found'),
+  release: N_ln('{num} release found', '{num} releases found'),
+  release_group: N_ln(
+    '{num} release group found',
+    '{num} release groups found',
+  ),
+  track: N_ln('{num} track found', '{num} tracks found'),
+};
+
+const noEntitiesText = {
+  recording: N_l('No recordings with this artist credit were found.'),
+  release: N_l('No releases with this artist credit were found.'),
+  release_group: N_l('No release groups with this artist credit were found.'),
+  track: N_l('No tracks with this artist credit were found.'),
+};
+
+type Props = {
+  +$c: CatalystContextT,
+  +artistCredit: $ReadOnly<{...ArtistCreditT, +id: number}>,
+  +entities: $ReadOnlyArray<CoreEntityT | TrackT>,
+  +entityType: string,
+  +page: string,
+  +pager: PagerT,
+};
+
+const EntityList = ({
+  $c,
+  artistCredit,
+  entities,
+  entityType,
+  page,
+  pager,
+}: Props): React.Element<typeof ArtistCreditLayout> => (
+  <ArtistCreditLayout $c={$c} artistCredit={artistCredit} page={page}>
+    <h2>
+      {expand2text(
+        headingsText[entityType](pager.total_entries),
+        {num: pager.total_entries},
```

Comment:
it'd be nice to import formatCount from root/statistics/utilities.js for these
```suggestion
        {num: formatCount($c, pager.total_entries)},
```

Reply:
Made the change in a second commit (both to these and the tag pages these are based on).

We probably should move that utility out of statistics, but I'm not sure where. If you have any suggestions, I can move it in a third commit.